HealthMonitoring.SelfHost/Content/Scripts/homeController.js

```diff
@@ -23,29 +26,68 @@
         var configFrequency = getConfigUpdatingFrequency($location);
 
         $scope.displayHint = function (evnt) {
-            if ($scope.statLegend == null) {
+            if ($scope.statLegend === null) {
```

Comment:
f.e. what we will get if statLegend will be 'undefined' ? 
in case of '===' we will get false, do we need this behavior